.release-it.yml

```diff
@@ -3,7 +3,8 @@ git:
   changelog: npx auto-changelog --stdout --commit-limit 0 --unreleased --template https://raw.githubusercontent.com/release-it/release-it/master/templates/changelog-compact.hbs
 github:
   release: true
-npm:
-  publish: false
+npm: false
 hooks:
-  after:bump: npx auto-changelog --package --starting-date 2022-02-15 --commit-limit 0 --tag-pattern 'v\d+.*'
+  after:bump: |
+    npx auto-changelog --package --starting-date 2022-02-15 --commit-limit 0 --tag-pattern 'v\d+.*';
+    echo "// This file is auto-generated by the release script\nexport const version = 'v${version}';" > src/version.ts
```

Comment:
Since src/version.ts is a versioned file, shouldn't it be commited after being generated?

Reply:
it is, this is how release-it works